docs/RFCs/20220915_commit_context.md

```diff
@@ -0,0 +1,180 @@
+- Feature: Commit context
+- Status: draft
+- Authors: @ijsong
+- Varlog Issue: #125
+- RFC PR:
+
+# Summary
+
+A large amount of empty commit contexts not only causes too slow recovery of
+log streams but also takes a quite large disk space. To solve these problems,
+this RFC proposes to store only the latest non-empty commit context rather
```

Comment:
I think we need latest commit context & latest non-empty commit context both.
Latest commit context is used for valid report version to trim commit result in metadata repository

Reply:
Yes, you're right. I'll fix it.

Reply:
@hungryjang 
Modified the draft to store the latest commit context regardless of emptiness.